tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -755,6 +755,140 @@ TEST(http_parser, alphabets)
 		"\r\n");
 }
 
+/**
+ * Test for case (in)sensitive matching of letters and special characters.
+ */
+TEST(http_parser, casesense)
+{
+	FOR_REQ("GET / HTTP/1.1\r\n"
+		"hOST: test\r\n"
+		/* We don't match open and closing quotes. */
```

Comment:
This is a copied comment, but unlike the place it was copied from, the line below has no `"` inside. I'd propose to delete the comment.